app/views/result_filters/location/new.html.erb

```diff
@@ -38,7 +38,7 @@
               <%= form.label :l_1, "By postcode, town or city", class: "govuk-label govuk-radios__label" %>
             </div>
             <div
-              class="govuk-radios__conditional <%= "govuk-radios__conditional--hidden" unless location_error? %>"
+              class="govuk-radios__conditional <%= "govuk-radios__conditional--hidden" unless request.params[:lq] || location_error? %>"
```

Comment:
Do we even need to include `location_error?`? shouldn't we always show the conditional fields if the parent radio button has been set.

Reply:
Is this the same change as whats coming in #92 ?

Reply:
Ah yes true